src/compas_blender/artists/frameartist.py

```diff
@@ -55,7 +61,10 @@ class FrameArtist(BaseArtist):
             artist.draw()
 
     """
-    def __init__(self, frame, collection=None, scale=1.0):
+    def __init__(self,
+                 frame: Frame,
+                 collection: Optional[bpy.types.Collection] = None,
+                 scale: Optional[float] = 1.0):
```

Comment:
as far as i know, `Optional` is only used when the default is `None`. so in this case it would just be `scale: float = 1.0`

Reply:
You are right, I'll fix this.